src/data/LedgerCache.hpp

```diff
@@ -83,109 +84,44 @@ class LedgerCache {
     std::unordered_set<ripple::uint256, ripple::hardened_hash<>> deletes_;
 
 public:
-    /**
-     * @brief Update the cache with new ledger objects.
-     *
-     * @param objs The ledger objects to update cache with
-     * @param seq The sequence to update cache for
-     * @param isBackground Should be set to true when writing old data from a background thread
-     */
```

Comment:
Is doxygen ok with the absence of documentation here because of `override`?

Reply:
Yeah, it takes the docs from the interface instead